scripts/SimpleQueries.sql

```diff
@@ -0,0 +1,16 @@
+-- Create a query to list out the event name and event data from the tblEvent table, with the most recent first.
+SELECT EventName, EventDetails, EventDate FROM dbo.tblEvent ORDER BY EventDate DESC;
+
+-- Create a query to list out the id number and name of the last 3 categories from the tblCategory table in alphabetical order
+SELECT * FROM dbo.tblCategory WHERE CategoryID > (SELECT COUNT(*) FROM dbo.tblCategory) - 3 ORDER BY CategoryName;
```

Comment:
I guess we have interpreted this differently. I've got this result 😁
![image](https://github.com/TheCSharpAcademy/CodeReviews.SQL.SimpleQueries/assets/108711476/8995fe88-a032-481e-9538-416eafe71ea0)

Reply:
I on the other hand got these result 😆 Idk why 
![image](https://github.com/TheCSharpAcademy/CodeReviews.SQL.SimpleQueries/assets/57032138/ec0c22a7-2f01-4f9a-aac8-0de6c0ab363e)

Reply:
Does alphabetical order mean sorted by the names or by the ids?

Reply:
I guess I will go with Id

Reply:
> Does alphabetical order mean sorted by the names or by the ids?

Although this isn't specified I would assume by the context that alphabetical will refer to text and not to numbers, but I'll approve the project anyways in compensation for my delay. And because it's (slightly) open to interpretation 😂